### ISSUE TITLE
Tweak to enable ‘More Space’ display zoom setting on 3rd & 4th gen 12.9 iPad Pro

### ISSUE THREAD

**Issue by Apple5576** [straight-tamago/misakaX#367]
I currently have extended monitor stage manager support enabled on my 12.9” 3rd gen iPad Pro via misakaX. 

I am trying to find a way of enabling the ‘More Space’ scaling option in display zoom settings, a feature restricted to the 5th gen and above 12.9” iPad Pro. I initially edited the MobileGestalt plist to spoof my device as a 5th gen model, but this did not reveal the setting. Does anyone know the relevant edit required in the MobileGestalt plist to reveal the ‘More Space’ display scaling option? Thank you

Settings → Display & Brightness → Display Zoom:
Larger Text
Default
More Space*

**Comment by BlankIsDumb** [comment]
Hey, this might be a bit too late, but you actually have to flip boolean `DeviceSupportsReverseZoom` key `sQwlfROu8fcD1Qwm8YJVeg` to true.

**Comment by Apple5576** [comment]
> Hey, this might be a bit too late, but you actually have to flip boolean `DeviceSupportsReverseZoom` key `sQwlfROu8fcD1Qwm8YJVeg` to true.

Thank you for this. I have just opened my MobileGestalt.plist in textwrangler on a Mac and can’t see this key in my iPad Pro 12.9 3rd gen file. Would I need to add this key into my plist manually? Please can you send me a gestalt.plist file with it in or the exact formatting I need to copy into my plist file. Thank you

**Comment by BlankIsDumb** [comment]
Yes, add the key in CacheExtra, I'm actually on 27b4 with mond tweaks so I can't export the mobilegestalt file, but you can just go ahead and add it under CacheExtra, key `sQwlfROu8fcD1Qwm8YJVeg` value `1` or `true` depending on your your tool, after that go ahead and respring and it should pop up under settings assuming everything goes correctly.
I actually had 5.6 sol work for like 2 hours to extract the 27b4 ipsw file, turns out that `AvailableDisplayZoomSizes` is actually a result of the function, and that function takes bool `DeviceSupportsReverseZoom` to actually compute it, also the settings app itself should already have the flag open to show the option, only one key is enough in my case, and so should yours. 
Sidenote: if you have changed your device's subtype into something else, change it back, because the dense/denser (more space) options uses your device's subtype and screen resolution to map the dense option to hardcoded resolutions, changing it may cause it to use the wrong resolution and who knows what happens then.
